frontend/views/pages/Home.vue

```diff
@@ -70,6 +113,9 @@ export default {
         // Enable focus on button and fix for firefox
         this.$nextTick(() => this.$refs[this.lastFocus].$el.focus())
       }
+    },
+    logout () {
+      sbp('state/vuex/dispatch', LOGOUT)
```

Comment:
It is a mere coincidence that the value for `LOGOUT` is the same as the name of the corresponding Vuex action.

This gives the impression that they are the same when they are not, and in the future it is possible they will diverge.

`LOGOUT`, along with every other event defined in `'@utils/events.js'` is only meant to be used by `okTurtles.events/emit` and `okTurtles.events/on`.

So this should be changed to the literal `'logout'` string.

The alternative is modifying `state.js` to define the Vuex action function names to use `LOGOUT` as well, using this kind of syntax:

```js
[LOGOUT]: async function (
  { dispatch, commit, state }: {dispatch: Function, commit: Function, state: Object}
) {
  // ...
```

etc.

That way they would be tightly bound and any changes to the definition of `LOGOUT` become future-proof.

I personally recommend keeping things simple for now and just using `'logout'` here.

Reply:
> I personally recommend keeping things simple for now and just using `'logout'` here.

Since `LOGOUT` is defined in a file called `events.js`, that is another reason to keep its usage limited to just events.